frontend/src/app/main/refs.cljs

```diff
@@ -232,23 +232,13 @@
   (l/derived #(get % id) workspace-page-objects))
 
 (defn objects-by-id
-  ([ids]
-   (objects-by-id ids nil))
-
-  ([ids {:keys [with-modifiers?]
-         :or { with-modifiers? false }}]
-   (let [selector
-         (fn [state]
-           (let [objects (wsh/lookup-page-objects state)
-                 modifiers (:workspace-modifiers state)
-                 ;; FIXME: Improve performance
-                 objects (cond-> objects
-                           with-modifiers?
-                           (gsh/merge-modifiers modifiers))
-                 xform (comp (map (d/getf objects))
-                             (remove nil?))]
-             (into [] xform ids)))]
-     (l/derived selector st/state =))))
+  [ids]
+  (let [selector
+        (fn [state]
+          (let [objects (wsh/lookup-page-objects state)
+                xform (comp (map (d/getf objects)) (remove nil?))]
```

Comment:
this can be simplified by using `keep` instead of map + filter nil?

Reply:
OK